src/router/movies.ts

```diff
@@ -0,0 +1,69 @@
+import express, {Request, Response} from 'express';
+import {Movie} from "../types/movie";
+import storage from '../storage/movie';
+import { v4 as uuidv4 } from 'uuid';
+import MovieService from "../service/movie";
+const router = express.Router();
+
+const getHandler = (req: Request, res: Response) => {
+    const { sortOrder, sortBy, limit, page } = req.query;
+    const movies = storage.getAll(sortOrder, sortBy, limit, page);
```

Comment:
If function need 4 or more args better to put it to object 
const movies = storage.getAll({ sortOrder, sortBy, limit, page });